types/cvss/tsconfig.json

```
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+          "es5"
+      ],
+      "allowSyntheticDefaultImports": true,
+      "esModuleInterop": true,
+      "noImplicitAny": true,
```

```suggestion
      "noImplicitAny": true,
```
please do not modify default config, unless there is a valid reason:
https://github.com/DefinitelyTyped/DefinitelyTyped#esmoduleinteropallowsyntheticdefaultimports

---

Sorry, I copied + pasted this from a time I had contributed previously